test/e2e/windows/framework.go

```diff
@@ -30,6 +32,18 @@ func SIGDescribe(text string, body func()) bool {
 			e2eskipper.SkipUnlessNodeOSDistroIs("windows")
 		})
 
+		// enable HostProcessContainers by default (it is Beta and on by default in beta in 1.23)
+		// this allows us to use host process containers in some of the tests but allows for
+		// someone that disables it to still run a subset of the tests
+		if framework.TestContext.FeatureGates == nil {
+			framework.TestContext.FeatureGates = map[string]bool{}
+			framework.TestContext.FeatureGates[string(features.WindowsHostProcessContainers)] = true
+		}
+		_, exists := framework.TestContext.FeatureGates[string(features.WindowsHostProcessContainers)]
```

Comment:
just check that this doesn't do what people use to think it does

https://kubernetes.slack.com/archives/C0EG7JC6T/p1643981225915189?thread_ts=1643970594.380149&cid=C0EG7JC6T

this featuregate field doesn't enable anything

Reply:
We are aware that it is just used as a flag for the e2e suite, it still requires the cluster to be configured properly but does allow disabling subset of the tests.  If I break the tests up as suggested, we won't need this since we can use a tag instead.